### PR TITLE
HTBHF-1528 Updated api to accept phone number. 

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/ClaimantTest.java [hunk 1]
```diff
@@ -196,4 +193,34 @@ void shouldFailToValidateClaimantWithoutPhoneNumber() {
...
+        //When
+        Set<ConstraintViolation<Claimant>> violations = validator.validate(claimant);
+        //Then
+        assertThat(violations).hasSingleConstraintViolation("invalid uk phone number, must be in +447123456789 format", "phoneNumber");
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {
+            "+447123456789",
+            "+44212345678", // non mobile numbers can be 10 digits
```
Comment (mikeSmith08) [inline api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/ClaimantTest.java]: I think you mean 9 digits here...

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimantDTO.java [hunk 2]
```diff
@@ -58,4 +58,10 @@
     @JsonProperty("address")
     @ApiModelProperty(notes = "The address of the claimant")
     private AddressDTO address;
+
+    @NotNull
+    @Pattern(regexp = "^\\+44\\d{9,10}$", message = "invalid uk phone number, must be in +44 format, e.g. +447123456789")
+    @JsonProperty("phoneNumber")
+    @ApiModelProperty(notes = "The claimant's phone number. May be used for verification purposes.")
```
Comment (mikeSmith08) [inline api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimantDTO.java]: Its probably worthwhile mentioning that is must be in UK format here.

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/ClaimantTest.java (hunk 1 above)
Comment (dwybourn) [reply api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/ClaimantTest.java]: Yep, updated.

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimantDTO.java (hunk 2 above)
Comment (dwybourn) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimantDTO.java]: Done.

**Pull request merged**